### ISSUE TITLE
[For June 2026] Introduce an alternative way to navigate the doc site (multiple ways)

### ISSUE THREAD

**Issue by helennickols** [ministryofjustice/moj-frontend#1897]
## What

On 15 Oct Eu-Hyung raised that there should be an alternative way to navigate items on the site. Site map was suggested by our FE dev as a way of approaching this. 

## Why

To improve the accessibility of the site. 

## Who needs to be involved

- Interaction Designer
- Content Designer
- Developer
- Accessibility Specialist

## Done

An alternative way has been developed and deployed.

**Comment by euhyung** [comment]
reason: [WCAG 2.4.5](https://www.w3.org/WAI/WCAG22/Understanding/multiple-ways)

**Comment by Asma-Ban** [comment]
The first iteration will be a site map
The second iteration could be a site search.

**Comment by helennickols** [comment]
@Asma-Ban let me know which date you'd like me to change this to and I can put it through. We can discuss in planning if easier.

**Comment by Asma-Ban** [comment]
@chrispymm @lewislankshear @helennickols to discuss sizing and next steps. To be decided whether we bring it to another cycle.

**Comment by helennickols** [comment]
@Asma-Ban Our accessibility specialist is happy with a Sept release. I'll change the date on the statement now.
